### ISSUE TITLE
Custom Severity support on Module

### ISSUE THREAD

**Issue by TechDufus** [PowershellGroup/Write-Log-Module#3]
Currently, you can run the following using the class:

```ps
using module ".\Class\Write-Log\Write-Log-Class.psm1"

$Log = [WriteLog]::New("C:\Example\mylog.log")
$Log.AddEntry("Testing","Test Severity")
```

Here we can see that we provide the Message 'Testing', and the Severity 'Test Severity'.

---------------------------------
In the module version, the `$Severity` parameter uses `[ValidateSet()]` and you can only select `info`, `warning`, or `error` and can't provide a custom severity.

---------------------------------

`SUGESSTED SOLUTION`:
My suggestion is instead of using `[ValidateSet()]`, we use a custom `[ArgumentCompleter]` to return these three (`info`, `warning`, or `error`) via tab completion, which is also provided with `[ValidateSet()]`, but allows for any value to be provided, supporting a custom severity.

**Comment by captainqwerty** [comment]
Excellent idea.  Implementing in the minor update version I am adding a few things to.

Thank you @matthewjdegarmo

**Comment by captainqwerty** [comment]
Resolved in version 1.1.0
